packages/graphback-core/src/db/dataMapper.ts

```diff
@@ -1,55 +1,36 @@
-import { ModelTableMapping } from './createModelTableMappings';
+import { ModelTableMap } from './buildModelTableMap';
 
-export interface DataMapping {
-    id?: TableID
+export interface TableDataMap {
+    idField?: TableID
     table?: string
     data?: any
     fieldMap?: any
 }
 
 export interface TableID {
     name: string
-    value: any
+    value?: any
 }
 
-export const mapDataToTable = (data: any, modelMap: ModelTableMapping): DataMapping => {
-    const fieldMap = modelMap.fieldMap;
+export const getDatabaseArguments = (modelMap: ModelTableMap, data?: any, fieldMap?: any): TableDataMap => {
+    const idField = modelMap.idField;
 
-    const dataMap: DataMapping = {
-        table: modelMap.tableName,
-        fieldMap
-    }
-
-    if (!fieldMap) {
-        return dataMap;
-    }
+    // TODO: Map fields to custom db names
 
-    dataMap.data = {};
-    for (const key of Object.keys(data)) {
-        const newKey = fieldMap[key] ? fieldMap[key] : key;
-        if (data[key]) {
-            dataMap.data[newKey] = data[key];
-        }
+    return {
+        idField: getTableId(idField, data),
+        data
     }
-
-    return dataMap;
 }
 
-export const mapDataFromTable = (modelName: string, data: any, modelsMap: ModelTableMapping[]): any => {
-    const modelMap = getModelMappingByName(modelName, modelsMap);
-    const fieldMap = modelMap.fieldMap;
-
-    const mappedData = {};
-    for (const key of Object.keys(fieldMap)) {
-        if (fieldMap[key] && data[fieldMap[key]]) {
-            const newKey = fieldMap[key];
-            mappedData[key] = data[newKey];
-        }
+function getTableId(idField: string, data: any = {}): TableID {
```

Comment:
I question value of this function. It do not check if idField is undefined. Or handle situation when data not exist (which means there is no chance to do query or mutation).